crabs/detection_tracking/train_model.py

```diff
@@ -116,7 +122,68 @@ def setup_trainer(self):
             limit_train_batches=self.limit_train_batches,
         )
 
-    def train_model(self):
+    def optuna_maximise_val_precision_recall(
```

Comment:
Can we call it objective, please? 
It's very confusing when it is not called objective. (that's why I prefer it to be in it's own file).
We have the explanation of what the function do in the docstring.